docs/UserGuide.md

```diff
@@ -13,11 +13,173 @@
 
 ## Features 
 
+## 1. Bookmark
+Now that more lessons are moving online, are you managing your links well? You might 
+want to save your time finding your zoom links in your email inbox and save all your 
+links in the bookmark feature to access them easily! The bookmark mode enables you to 
+compile all your links in one destination and even categorise them into different groups. 
+After entering the bookmark mode (refer to General Command 2), use the following commands to 
+navigate around bookmark mode. This section contains 8 commands : 1.1, 1.2, 1.3, 1.4, 1.5, 
+1.6, 1.7 and 1.8. 
 
+###1.1 Viewing bookmarks in each category : 
```

Comment:
There should be a space after the # in order to display the heading properly (### 1.1 Viewing bookmarks in each category:)